### DISCUSSION TITLE
git commit 的时候报错 Unexpected unknown at-rule "@tailwind"

### DISCUSSION THREAD

**Discussion by Cooper1886** [umijs/umi#11210] [Q&A]
![image](https://github.com/umijs/umi/assets/107901138/2398758f-e7ac-4070-9643-014ce3fe7e71)
![image](https://github.com/umijs/umi/assets/107901138/f36ceec0-691f-4a61-ae3c-246c25e4a8bb)
请教这个报错要如何解决呢

**Comment by fz6m** [comment]
参考一下 [这个文章](https://scottspence.com/posts/stylelint-configuration-for-tailwindcss) 修改下 stylelint 配置吧。

**Reply by Cooper1886** [reply]
感谢

**Comment by Cooper1886** [comment]
感谢
以下是配置
.stylelintrc.js
```javascript
module.exports = {
  extends: require.resolve('@umijs/max/stylelint'),
  rules: {
    'at-rule-no-unknown': [
      true,
      {
        ignoreAtRules: [
          'tailwind',
          'apply',
          'variants',
          'responsive',
          'screen',
        ],
      },
    ],
  },
};
```
